Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -569,15 +583,22 @@ bool ReflectometryReductionOneAuto::processGroups() {
     secondTransG = boost::dynamic_pointer_cast<WorkspaceGroup>(secondTransWS);
 
     if (!secondTransG)
-      alg->setProperty("SecondTransmissionRun", secondTrans);
-    else if (group->size() != secondTransG->size())
-      throw std::runtime_error("SecondTransmissionRun WorkspaceGroup must be "
```

Comment:
This check is still valid if you are NOT doing PolarizationCorrection. So make sure it still runs the check if you are NOT doing PolarizationCorrection.